sshclient.go

```diff
@@ -106,6 +106,11 @@ func (c *Client) Close() error {
 	return c.client.Close()
 }
 
+// Close closes the underlying client network connection.
+func (c *Client) NativeClient() *ssh.Client {
```

Comment:
I think it is more appropriate to call it `UnderlyingClient`.
In programming,  `Native` doesn’t seem to mean this.
What do you think so?

Reply:
Yes, I think so.